src/routes/api/user/server.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, test, vi } from 'vitest';
```

Comment:
Three simple tests in this file which gives 100% coverage on the `user` route. I decided to colocate the test file with the code which makes it easy to find them.